src/features/productDetails/ProductDetails.tsx

```diff
@@ -38,10 +46,27 @@ const ProductDetails = () => {
                         <h2 className="text-3xl">{product?.title}</h2>
                         <h2 className="text-2xl">{formattedPrice(product?.price ?? 0)}</h2>
                         <div className="flex flex-wrap w-full gap-2 justify-center">
-                            <button className="w-full md:w-2/5 btn" onClick={() => addItemToCart(product?.id)}>Add to
-                                Cart
+                            <button
+                                className="w-full md:w-2/5 btn btn-primary"
+                                onClick={() => addItemToCart(product?.id)}
+                            >
+                                Add to Cart
                             </button>
-                            <button className="w-full md:w-2/5 btn btn-neutral">Wishlist</button>
+                            {isInWishlist ? (
```

Comment:
An alternative to this logic can be the following (for fewer lines of code):

```
 <button className={`w-full md:w-2/5 btn ${isInWishlist ? "btn-neutral" : ""}`}
onClick={isInWishlist ? () => removeFromWishlist(product?.id) : () => addToWishlist(product?.id)}
>
{ isInWishlist ? "Remove from Wishlist" : "Add to Wishlist"}
</button> 
```

Just a suggestion, doesn't really have to be changed